backends/advanced/src/advanced_omi_backend/config.py

```
@@ -0,0 +1,111 @@
+                logger.info(f"Created diarization config from template at {config_path}")
+            except Exception as e:
+                logger.warning(f"Could not copy template to {config_path}: {e}")
+    
+    # Load from file if it exists
+    if config_path.exists():
+        try:
+            with open(config_path, 'r') as f:
+                _diarization_settings = json.load(f)
+                logger.info(f"Loaded diarization settings from {config_path}")
+                return _diarization_settings
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)